File: Modul-2/Soal-3/PRAK203-2410817220023-CLARISSADHEAALLISYA.c
#include <stdio.h>

int main () {
    //test case 1
    float a, b, i, j, x, y;

    scanf("%f %f %f %f %f %f", &a, &b, &i, &j, &x, &y);
    printf("%.3f", (a - b) * (i / j) - (x + y));
    
    //test case 2
    float a2, b2, i2, j2, x2, y2;
    
    scanf("%f %f", &a2, &b2);
    scanf("%f %f", &i2, &j2);
    scanf("%f %f", &x2, &y2);
    printf("%.3f", (a2 - b2) * (i2 / j2) - (x2 + y2));
    return 0;
} 

File: Modul-2/Soal-3/PRAK203-2410817220023-CLARISSADHEAALLISYA.py
# test case 1
a, b, i, j, x, y = map(float, input().split())
hasil = (a - b) * (i / j) - (x + y)
print(f"{hasil:.3f}")

# test case 2
print("")
a2, b2 = map(float, input().split())
i2, j2 = map(float, input().split())
x2, y2 = map(float, input().split())
hasil1 = (a2 - b2) * (i2 / j2) - (x2 + y2)
print(f"{hasil1:.3f}") 